services/src/main/scala/cromwell/services/keyvalue/impl/KeyValueDatabaseAccess.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.services.keyvalue.impl
+
+import akka.actor.ActorSystem
+import cromwell.core.ExecutionIndex._
+import cromwell.util.DatabaseUtil._
+import cromwell.core.{JobKey, WorkflowId}
+import cromwell.database.Database
+import wdl4s.Scope
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait KeyValueDatabaseAccess { this: Database =>
+
+  def getExecutionInfoByKey(workflowId: WorkflowId, call: Scope, attempt: Int, key: String)
+                           (implicit ec: ExecutionContext): Future[Option[Option[String]]] = {
+    databaseInterface.getExecutionInfoByKey(workflowId.toString, call.fullyQualifiedName, attempt, key)
```

Comment:
Is this really separating things out if it's just redirecting back to a shared `databaseInterface` method?